applications/luci-app-softether/Makefile

```diff
@@ -0,0 +1,15 @@
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=SoftEther Status
+LUCI_DEPENDS:=+softethervpn5-client
+LUCI_PKGARCH:=all
+
+PKG_LICENSE:=Apache-2.0
+
+include ../../luci.mk
+
+# call BuildPackage - OpenWrt buildroot signature
```

Comment:
Can you add pictures in the overview of the pr, how that looks?

Reply:
Also please change the commit message to `luci-app-softether: add basic softether status page`.

Reply:
done